databricks/koalas/tests/test_stats.py

```diff
@@ -73,13 +73,24 @@ def test_stat_functions_multiindex_column(self):
         self._test_stat_functions(pdf, kdf)
 
     def test_stat_functions_with_no_numeric_columns(self):
+        # TODO: pandas support datetime64 or datetime64tz dtypes for `std` from pandas 1.2
+        # to return Timedelta Series which is Koalas currently cannot support.
+        # pdf = pd.DataFrame(
+        #     {
+        #         "A": pd.date_range("2020-01-01", periods=3),
+        #         "B": pd.date_range("2021-01-01", periods=3),
+        #     }
+        # )
+        # kdf = ks.from_pandas(pdf)
```

Comment:
Shall we file an issue for that instead of comment out here?

Reply:
Sounds good. Let me file up to the issue. Thanks :)